src/components/RecipientAddressRow.tsx

```diff
@@ -0,0 +1,34 @@
+import { ExternalLinkIcon } from '@chakra-ui/icons'
+import { HStack, Icon, Link } from '@chakra-ui/react'
+import type { Asset } from '@shapeshiftoss/types'
+
+import { Row } from '@/components/Row/Row'
+import { RawText, Text } from '@/components/Text'
+import { middleEllipsis } from '@/lib/utils'
+
+type RecipientAddressRowProps = {
+  sellAsset: Asset
+  recipientAddress: string | undefined
+}
+
+export const RecipientAddressRow = ({ sellAsset, recipientAddress }: RecipientAddressRowProps) => {
```

Comment:
All for making this reusable, nice one! 
Can we make this more dumb however and only pass `explorerAddressLink` in vs. the full `Asset`? 🙏🏽